styles.css

```
@@ -1332,3 +1332,107 @@
+	border-radius: var(--radius-s);
+}
+
+.mermaid-flow-feedback-notes-title {
+	margin: 0 0 var(--size-4-2);
+	font-size: var(--font-ui-medium);
+}
+
+.mermaid-flow-feedback-notes-body {
+	margin: 0 0 var(--size-4-2);
+	white-space: pre-wrap;
+	word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)